locales/de.json

```diff
@@ -247,25 +247,25 @@
   "app.auctions.search.option.sort.priceAsc": "Preis (aufsteigend)",
   "app.auctions.search.option.sort.priceDesc": "Preis (absteigend)",
   "app.auctions.search.nemesis.label.quirk": "Marotte",
-  "app.auctions.search.nemesis.placeholder.quirk": "Any",
+  "app.auctions.search.nemesis.placeholder.quirk": "Beliebig",
```

Comment:
The translation of "Any" was hard. I chose "Beliebig". Some alternatives: "keine Einschränkungen", "Egal", "Alle"
If you have a better suggestion, go ahead.